src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -38,24 +42,51 @@ instance TypeHasDoc Storage where
   typeDocMdDescription =
     "Contract's storage holding a big_map with all balances and the operators."
 
+instance HasFieldOfType Storage name field =>
+         StoreHasField Storage name field where
+  storeFieldOps = storeFieldOpsADT
+
+type StorageC store =
+  StorageContains store
+    [ "sAdmin" := Address
+    , "sPendingOwner" := Maybe Address
+    , "sLedger" := Ledger
+    ]
+
+-- | Parameter of the BaseDAO contract
+data Parameter
+  = Call_FA2 FA2.Parameter
+  | Transfer_ownership TransferOwnershipParam
+  | Accept_ownership AcceptOwnershipParam
+  deriving stock (Generic, Show)
+  deriving anyclass (IsoValue)
+
+instance ParameterHasEntrypoints Parameter where
+  type ParameterEntrypointsDerivation Parameter = EpdDelegate
+
+type TransferOwnershipParam = ("newOwner" :! Address)
+type AcceptOwnershipParam = ()
```

Comment:
`TransferOwnershipParam` is more or less ok, but `AcceptOwnershipParam` only complicates code, IMO, using `()` directly would be simpler.

Reply:
Changed